truffle-config.js

```diff
@@ -62,5 +62,11 @@ module.exports = {
       gasPrice: 20,
       showTimeSpent: true
     }
+  },
+  plugins: [
+    'truffle-plugin-verify'
```

Comment:
@Velenir I tried using `verify-on-etherscan` but ran into the following error:

```
Error: There is a cycle in the dependency graph, can't compute topological ordering. Files:
        contracts/libraries/SnappBaseCore.sol
        openzeppelin-solidity/contracts/token/ERC20/ERC20.sol
        /Users/felixleupold/Gnosis/dex-contracts/node_modules/openzeppelin-solidity/contracts/GSN/Context.sol
        /Users/felixleupold/Gnosis/dex-contracts/node_modules/openzeppelin-solidity/contracts/token/ERC20/IERC20.sol
        /Users/felixleupold/Gnosis/dex-contracts/node_modules/openzeppelin-solidity/contracts/math/SafeMath.sol
        @gnosis.pm/solidity-data-structures/contracts/libraries/IdToAddressBiMap.sol
        @gnosis.pm/solidity-data-structures/contracts/libraries/Merkle.sol
        /Users/felixleupold/Gnosis/dex-contracts/contracts/SnappBase.sol
        openzeppelin-solidity/contracts/ownership/Ownable.sol
        /Users/felixleupold/Gnosis/dex-contracts/contracts/libraries/SnappBaseCore.sol
    at getSortedFilePaths (/Users/felixleupold/Gnosis/dex-contracts/node_modules/truffle-flattener/index.js:95:13)
```

Reply:
It's an issue with `truffle-flattener` nomiclabs/truffle-flattener#14

Reply:
I added a new npm script that only verifies stablex. 
This avoids this issue, as only the snappAuction code has a cyclic dependency.